sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2SQLSuite.scala

```diff
@@ -1180,7 +1182,7 @@ class DataSourceV2SQLSuite
             val tableCatalog = catalog("testcat").asTableCatalog
             val identifier = Identifier.of(Array(), "reservedTest")
             assert(tableCatalog.loadTable(identifier).properties()
-              .get(TableCatalog.PROP_LOCATION) == "foo",
+              .get(TableCatalog.PROP_LOCATION) == makeQualifiedLocationPath("foo"),
```

Comment:
What's the real value?

Reply:
"file:" + WAREHOUSE_PATH + "/foo". eg: `file:/D:/code/bigdata/spark/spark-warehouse/org.apache.spark.sql.connector.DataSourceV2SQLSuite/foo`

Reply:
then how about we check `path.startsWith("file:") && path.endsWith("foo")`?

Reply:
OK.